snapshot/store/store.go

```diff
@@ -10,7 +10,12 @@ import (
 )
 
 var DefaultTTL time.Duration = time.Hour * 24 * 180 //180 days. If zero, no ttl will be applied by default.
-const DefaultTTLKey string = "x-scoot-expires"      //the primary use for this is communicating ttl(RFC1123) over http.
+const (
+	// DefaultTTLKey is primarily used for communicating ttl(RFC1123) over http.
+	DefaultTTLKey string = "x-scoot-expires"
+	// DefaultTTLFormat is to specify the format for parsing ttl(RFC1123).
+	DefaultTTLFormat string = time.RFC1123
+)
 
 // Stores should generally support TTL, at this time only httpStore implements it.
```

Comment:
as long as you're in here, would you mind editing this comment / adding one to TTLConfig?

Reply:
Done.